src/Uno.UI/UI/Xaml/Controls/ProgressRing/ProgressRing.iOS.cs

```diff
@@ -50,22 +50,29 @@ private static void OnForegroundChanged(DependencyObject dependencyObject, Depen
 			}
 		}
 
-        private static void OnIsActiveChanged(DependencyObject dependencyObject, DependencyPropertyChangedEventArgs args)
-        {
-            var progressRing = dependencyObject as ProgressRing;
-            var isActive = args.NewValue as bool?;
+		partial void OnVisibilityChangedPartial(Visibility oldValue, Visibility newValue)
+		{
+			SetIsActive(newValue == Visibility.Visible && IsActive);
+		}
+
+		private static void OnIsActiveChanged(DependencyObject dependencyObject, DependencyPropertyChangedEventArgs args)
+		{
+			(dependencyObject as ProgressRing)?.SetIsActive(args.NewValue as bool?);
+		}
 
-            if (progressRing != null && isActive != null)
-            {
-                if (isActive.Value)
-                {
-                    progressRing.StartAnimating();
-                }
-                else
-                {
-                    progressRing.StopAnimating();                    
-                }
-            }
-        }
-    }
+		private void SetIsActive(bool? isActive)
```

Comment:
There's no need for it to be a nullable type, `IsActive` is just a `bool`.

Reply:
Done